### PR TITLE
feat: Migrate wdio 'SendEthMultisig' test to Detox

### PR DESCRIPTION
**Pull request by sleepytanya**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/coding_guidelines/CODING_GUIDELINES.md)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add the appropiate QA label when dev review is completed
  - `No QA/E2E only`: PR does not require any manual QA effort. Prior to merging, ensure that you have successful end-to-end test runs in Bitrise. 
5. Add `QA Passed` label when QA has signed off (Only required if the PR was labeled with `needs-qa`)

**Description:**

This PR converts Appium E2E test to Detox.

**E2E:** https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/6e6f2584-baac-48a3-bf8c-f4d80b60e2ca?tab=workflows

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

fixes #???

**Checklist**

* [NA ] There is a related GitHub issue
* [NA] Tests are included if applicable
* [x] Any added code is fully documented

### CODE REVIEW

**Review by chrisleewilcox — COMMENTED** [review]
@sleepytanya please run the `pr_smoke_e2e_pipelin` workflow on bitrise for you branch before dev review and provide the link to the bitrise test results in this PR.

File: e2e/specs/confirmations/send-eth-multisig.spec.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    await SendView.inputAddress(MULTISIG_ADDRESS);
+    await SendView.tapNextButton();
+
+    await AmountView.typeInTransactionAmount(AMOUNT_TO_SEND);
+    await AmountView.tapNextButton();
+
+    await TransactionConfirmationView.tapConfirmButton();
+    await TabBarComponent.tapActivity();
+
+    await TestHelpers.checkIfElementByTextIsVisible(AMOUNT_TO_SEND + ' ETH');
```
Comment (cortisiko) [inline e2e/specs/confirmations/send-eth-multisig.spec.js:54]: Can you use the 'ETH' string from the `en.json` in the locales folder? The string is in the `unit` object.

**Review by cortisiko — CHANGES_REQUESTED** [review]
@sleepytanya code looks good, I left one small comment.

**Review by cortisiko — APPROVED** [review]
Looks good @sleepytanya

**Pull request merged**